sdcm/cluster.py

```diff
@@ -1653,10 +1657,18 @@ def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits=12):
         self.config_setup(murmur3_partitioner_ignore_msb_bits=murmur3_partitioner_ignore_msb_bits)
         self.start_scylla()
 
-        resharding_finished = wait.wait_for(func=self._resharding_finished, step=5, timeout=3600,
-                                            text="Wait for re-sharding to be finished")
+        resharding_started = wait.wait_for(func=self._resharding_status, step=5, timeout=3600,
```

Comment:
Is 3600 seconds too big for the resharding start?

Reply:
Yes, it's big, but we decided to set it "for sure". It's expected be started fast, so it won't wait so long time